docs/api/carts.rst

```diff
@@ -0,0 +1,764 @@
++-------------------+----------------------------------------------------------------------+
+| adjustments       | List of adjustments related to item                                  |
++-------------------+----------------------------------------------------------------------+
+| adjustments_total | Sum of all item adjustments                                          |
++-------------------+----------------------------------------------------------------------+
+| variant           | Default variant serialization                                        |
++-------------------+----------------------------------------------------------------------+
+
+Each cart item unit will be build as follows:
+
++-------------------+----------------------------------------------------------------------+
+| Field             | Description                                                          |
++===================+======================================================================+
+| id                | Id of cart item unit                                                 |
++-------------------+----------------------------------------------------------------------+
+| adjustments       | List of adjustments related to unit                                  |
++-------------------+----------------------------------------------------------------------+
+| adjustments_total | Sum of all units adjustments                                         |
++-------------------+----------------------------------------------------------------------+
+
+And each adjustment will be build as follows:
+
++--------+--------------------------------------------------------+
+| Field  | Description                                            |
++========+========================================================+
+| id     | Id of cart item unit                                   |
++--------+--------------------------------------------------------+
+| type   | Type of an adjustment (E.g. *order_promotion* or *tax* |
++--------+--------------------------------------------------------+
+| label  | Label of adjustment                                    |
++--------+--------------------------------------------------------+
+| amount | Amount of adjustment                                   |
++--------+--------------------------------------------------------+
+
+.. _Customer detailed serialization: http://docs.sylius.org/en/latest/api/customers.html#customers-api
+.. _Default channel serialization: http://docs.sylius.org/en/latest/api/channels.html#channels-api
+
+.. note::
+
+    If it is still confusing to you, you should read more about `Carts (Orders)`_ and `Adjustments`_ first.
+
+.. _Carts (Orders): http://docs.sylius.org/en/latest/components/Order/models.html#order
+.. _Adjustments: http://docs.sylius.org/en/latest/book/orders/adjustments.html
+
+Creating a Cart
+---------------
+
+Definition
+..........
+
+.. code-block:: text
+
+    POST /api/v1/carts/
```

Comment:
Shouldn't we give a full command? with `$ curl` part? (in `.. code-block:: bash`)